### ISSUE TITLE
Creating better instructions for how to label pedestrian signals

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#3240]
How many labels should I be applying here? It's not totally clear to me. Should I be labeling the "output posts" as well as the "input/button posts" in those cases when they are separate?

![image](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/1621749/3ea568a2-5d7c-46cb-9a97-31992d1520a8)

![image](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/1621749/d998064b-726d-418e-b824-ab514ab558a3)

![image](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/1621749/33a1ff1d-5396-45ed-a23c-622a37866db7)

**Comment by jonfroehlich** [comment]
Here's the slide deck.

[PedestrianSignals.pptx](https://github.com/ProjectSidewalk/SidewalkWebpage/files/11548288/PedestrianSignals.pptx)

**Comment by misaugstad** [comment]
Very good questions! We should continue to discuss with Yochai and Devon while working on the labeling guide!

**Comment by misaugstad** [comment]
A possible set of tags to get at the information we want, while also making it clearer to users how to label:
* 1 button
* 2 buttons (can't pick both)
* 1 walk light
* 2 walk lights (can't pick both)

We should iterate on the language for counting the number of directions for which a single pole has walk signal lights.

I do think that counting the number of buttons/lights on each corner will make this more clear for users, and will get us the information we want most of the time.

**Comment by yeisenberg** [comment]
This seems like a good direction to go. Also, got some feedback from a colleague that one key element to focus on could be whether the rater visually sees an arrow on the button and that this can be a good indicator in nearly all cases that this is an APS. He shared this link - https://polara.com/ids-ins.  In the Chicago area, we see a ton that do not have this and that i've noticed don't seem to have the full APS capabilities of audio signals.

**Comment by jonfroehlich** [comment]
Nice. Thanks Yochai!

**Comment by yeisenberg** [comment]
HI there, at our workshop, this was brought up as an important feature as we know.  Thought I'd post again to restart this thrust to put the 'audio signal' tag as part of pedestrian signals that are key for people who are blind/low vision. thanks

**Comment by yeisenberg** [comment]
After some back and forth with Mikey on Slack am adding to this conversation. Based on that and this link related to new PROWAG - https://polara.com/guide/prowag-accessibility-requirements. Taking away from it 
The discussion has to do with whether to have all 4 tags about the 1 or 2 buttons, or just the tactile button tagss? A key  question is whether knowing about non-tactile-audio buttons is important?

-     1 button
-     2 buttons (can't pick both)
-    1 tactile-Audio button
-    2 tactile-Audio  button (can't pick both)
-     1 walk light
-     2 walk lights (can't pick both)
- hard to reach button
- obstruction to pushing button

**Comment by jonfroehlich** [comment]
I think we should have:
- APS (with a nice hover image to explain)
- Walk light (again, nice hover image)
- Hard to reach button (if someone clicks this, does "has button" get auto-selected? or maybe we just deal with this in post-hoc analysis)

I'm not confident that having people count is a good idea; it also introduces ambiguity about the other tags. For example, if I put "2 buttons" then what do the other tags correspond to (e.g., button hard to reach, APS).

**Comment by misaugstad** [comment]
@jonfroehlich and I talked a bit more after this. Here's what I think we settled on for the list of tags:
* has button (unchanged)
* button waist height (unchanged)
* APS (unchanged, but I need to unhide it) -- @yeisenberg lmk if you have a different preferred name for this tag
* hard to reach button (this is new, ideally have it auto-select "has button")

And then I need to add example images for all of them.

**Comment by yeisenberg** [comment]
Great discussion! Wondering though, since all the other labels have tags that would be considered 'negative' or 'missing' whether we should make all of these that way too.
Suggested approach:
- missing tactile-audible button
- button is too high
- button is hard to reach (maybe makes sense to combine with button is too high?) 
- missing walk signal (haven't looked at Prowag but maybe we should to know if this required)

**Comment by misaugstad** [comment]
>button is hard to reach (maybe makes sense to combine with button is too high?)

I really like that!

>missing tactile-audible button

I think it's worth having this one in the positive instead of negative because of the knowledge required to know when this is missing. If users don't know what this means and therefore don't label it, we'd assume that all the pedestrian signal are tactile-audible, which I don't think is what we want. For now, I think I'd rather have people who know what they're doing choose to add a tag!

>missing walk signal (haven't looked at Prowag but maybe we should to know if this required)

I think that we don't need to worry about adding this yet. What would it even look like to have a pedestrian signal without the walk signal? Just a button that doesn't do anything or only has sound?

So in summary, we're going with the following for now:
* has button (unchanged)
* tactile-audible button (changed from "APS", need to unhide it)
* hard to reach button (replacing "button waist height")

**Comment by yeisenberg** [comment]
Thanks for the updates Mikey!! For signals with two buttons, would we suggest that the auditor place two tags then?

**Comment by misaugstad** [comment]
I think that as of now, we are punting on figuring out whether there are one or two buttons. And when we do clustering, the two labels would be clustered together into only one anyway, so I think that we just add label even if there are multiple buttons!

**Comment by jonfroehlich** [comment]
It's funny, I was labeling the other day, and perhaps because this was top-of-mind, felt like it was relevant to have a "has two buttons" tag (but I still don't see how can operationalize that given that the other tags assume only one button like "missing tactile-audible button" and "button is too high"—those labels begin to break down when we have the "has two buttons" tag because it creates ambiguity). Hmm....

So that does just siggest having the auditor place two tags: one for each button.

**Comment by misaugstad** [comment]
IMO it would be better to have the ambiguity of not knowing which buttons are being referred to with the tags than it would be to train users to label the same pole multiple times because there are multiple buttons.

I'm basing that off the assumption that on a single pole it would be almost always be the case that both the buttons would have the same design. Not sure if others have a sense of how accurate that is.

**Comment by jonfroehlich** [comment]
True. I think you're right. (And, in practice, I've found that the same buttons are designed on the same pole...)

So, do we want to add "has two buttons" then?

**Comment by misaugstad** [comment]
I'd recommend replacing "has button" with "one button" and then adding a "two buttons" tag. And selecting one would automatically deselect the other.

**Comment by jonfroehlich** [comment]
Nice, yes! Thank you.

**Comment by misaugstad** [comment]
Great, so the new plan (switching from singular to plural for the other tags):
* one button
* two buttons (can't have this and "one button" selected)
* hard to reach buttons
* tactile-audible buttons (Chicago only)

This work for you @yeisenberg ?

**Comment by yeisenberg** [comment]
Looks good to me! what is the thought with Chicago only? like a testing phase?

**Comment by jonfroehlich** [comment]
I think we should deploy it everywhere (at least in US)

**Comment by misaugstad** [comment]
That's fair, now that I've added an example image! :grin:
